src/i18n/de.js

```diff
@@ -0,0 +1,43 @@
+export default {
+    header: {
+        slogan: 'Die Zukunft der Web-Entwicklung.'
+    },
+    callToAction: {
+        start: 'Jetzt loslegen',
+        join: 'Am Chat teilnehmen'
+    },
+    nav: {
+        code: 'Code',
+        cloud: 'Cloud',
+        docs: 'Doku'
+    },
+    bio: {
+        first: 'Asynchrone, Ereignis-gesteuerte Architektur die auf Apples <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> basiert und <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">höchste Performance</a> liefert.',
```

Comment:
I understand, that `Ereignis-gesteuerte` sets a focus on the `Ereignis`-Scheme, but I think this word looks weird. So I'd rather use the english `event`, so `Event-gesteuerte` or `ereignisgesteuerte`.

Reply:
And there is a comma missing: `...Architektur, die...`

Reply:
I went with `Event-gesteuerte` for now, but I think both your suggestions are very good.